### PR TITLE
"Controlled" property for logic

### PR DESCRIPTION
**Pull request by summetdev**
Closes Anuken/Mindustry-Suggestions#945.
Suggested by @Slava0135.

### CODE REVIEW

File: core/src/mindustry/logic/LAccess.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
     team,
     type,
     flag,
+    bounded,
```
Comment (summetdev) [inline core/src/mindustry/logic/LAccess.java:33]: Possible alternatives are: `bound`, `isBound`, `isBounded`.

**Comment by Quezler** [comment]
it doesn't seem very descriptive, what about `controlled` or something a little more self explanatory ?

**Comment by Slava0135** [comment]
> it doesn't seem very descriptive, what about `controlled` or something a little more self explanatory?
> (personally it reminds me a lot of boundaries like (map) borders or the max radius of the processor)

agreed

**Comment by summetdev** [comment]
@Quezler changed, thanks.

**Comment by Anuken** [comment]
This should be used for buildings as well. Turrets (and routers?) should return 1 if they are controlled.

**Comment by summetdev** [comment]
@Anuken now it works with blocks too.

**Pull request merged**